src/Diagnostics.RuntimeHost/Controllers/WorkerApp/WorkerAppController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Threading.Tasks;
+using Diagnostics.ModelsAndUtils.Models;
+using Diagnostics.ModelsAndUtils.Models.ResponseExtensions;
+using Diagnostics.RuntimeHost.Models;
+using Diagnostics.RuntimeHost.Utilities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Newtonsoft.Json;
+using Microsoft.CSharp.RuntimeBinder;
+
+namespace Diagnostics.RuntimeHost.Controllers
+{
+    [Authorize]
+    [Produces("application/json")]
+    [Route(UriElements.WorkerAppResource)]
+    public class WorkerAppController : DiagnosticControllerBase<WorkerApp>
```

Comment:
Do we need a separate Controller class for this? Doesn't the existing ARM Resource Controller work ?